### PR TITLE
put training and validation loss in the same graph in tensorboard

### PR DESCRIPTION
**Pull request by txzhao**
When babysitting the training process, gap between training and validation accuracy indicates amount of over-fitting. Therefore, I tried out a way mentioned in a stackoverflow thread to put training and validation loss in the same graph using tensorboard. 

Reference: 
https://stackoverflow.com/questions/34471563/logging-training-and-validation-loss-in-tensorboard

Now, it looks like this:
losses

### CODE REVIEW

**Comment by shekkizh** [comment]
@txzhao Thanks for the addition. I'm away from my system. Will look into the PR and merge asap.

**Comment by txzhao** [comment]
hey @shekkizh, may I ask if there is something wrong with my commits? It's been a month and still it's not merged :(

**Comment by varungupta31** [comment]
Hey @txzhao 
I ran the training or a few epochs, and was visualizing the metrics using tensorboard.
But i'm only getting one curve there 'entropy' 

image

I want to log and visualize other metrics like - mIOU, accuracy, etc.
Will doing more epochs add more curves in tensorboard? if yes, then does those curved include accuracy, etc.? 

Please help me out, if possible.

TIA
